grpc/core/src/main/java/module-info.java

```diff
@@ -20,6 +20,14 @@
 module io.helidon.grpc.core {
 
     requires io.helidon.common;
+    requires io.helidon.http;
+    requires io.helidon.common.context;
+    requires io.helidon.config;
+    requires io.helidon.config.metadata;
```

Comment:
Config metadata can only be `requires static`, if used at all. 
New configurable types should be built using builder API, and that does not use config metadata